src/store/sidebarStore.js

```diff
@@ -21,7 +21,7 @@
  */
 
 const state = {
-	show: true,
+	show: false,
```

Comment:
I remember that we discussed in the past that we could close it by default once the conversation name was displayed in the main content..
Mind sharing the reasoning behind why you don't want it closed by default?

Reply:
Microsoft teams, Slack, Whatsapp, Telegram, Signal, Google chat and I'm sure many more don't show their "details" sidebar by default.
We have a chance to really simplify our UI here and I think we should take it

Reply:
My 2 reasons are:
1. In all my calls with non Nextcloud people I hear how they suffer to find things. Folding away the complete sidebar into an undestinct Burger+Person on top of it icon is not going to improve that.
2. 2700px of whitespace (4k screen - chat width - left sidebar) instead of being able to see useful information with the participants list

> Microsoft teams, Slack, Whatsapp, Telegram, Signal, Google chat and I'm sure many more don't show their "details" sidebar by default.

There is a huge difference: those are dedicated clients which you don't run on full screen all the time oposed to your browser, so there is less whitespace.
And our conversation info doesn't show the participants and if "everyone does it" is your reasoning, people would assume they can find it there so we would need to duplicate it there too which I'm even more against.

Reply:
On point 1:
I don't think it's a good principle to just show the whole sidebar just because we're worried that users might not find, If there's a discoverability problem, we can tackle it in more appropriate ways. The sidebar doesn't need to be toggled by the hamburger button only, it can be opened by several buttons in different contexts. E.g. There could be a participants action button in the three dot menu in the topbar that opens the sidebar with the participants tab focused;

On point 2: 
- I don't think that your screen is a good reference on which to build talk's default experience. 
A very very tiny fraction of users have such resolution. Most users are still on 1080p displays, on which windows still scales the display 125% to 150% on a  14" - 15" laptops. This means that by default our chat would be squeezed to make room for the sidebar by default. This worsen the chat experience to show information that is redundant to many users.
- Not saying that we shouldn't handle huge resolution by creating special exceptions for those. Just saying that it shouldn't be our default.

> There is a huge difference: those are dedicated clients

Many of those behave similarly in their web UIs

> And our conversation info doesn't show the participants and if "everyone does it" is your reasoning, people would assume they can find it there 

Not all of those apps have a list of participants in the topbar. And btw, you know my aim is to simplify the UI and make talk easier and easier to use, not just blindly copy what others do. 

One further thing that would not be ideal if we show it by default, is that unless we remove the whole header in the sidebar, we'd have a bunch of duplicated info there shown by default. See screenshot below: 

![Screenshot from 2021-05-11 09-01-14](https://user-images.githubusercontent.com/26852655/117780404-7fae3200-b237-11eb-8bae-6b143d8378d9.png)

Also notice the squeezed chat there ^

Reply:
There is by far no squeezed chat with full HD.
![Bildschirmfoto von 2021-05-11 10-20-20](https://user-images.githubusercontent.com/213943/117783258-aa51b800-b242-11eb-98c7-71c6deea5f50.png)

Reply:
> Most users are still on 1080p displays, on which windows still scales the display 125% to 150% on a 14" - 15" laptops.